README.md

```diff
@@ -62,6 +62,30 @@ gcp-scanner --help
 There is a docker build file if you want to run the scanner from a container:
 `docker build -f Dockerfile -t sa_scanner .`
 
+### Setup of tool(gcp_scanner)
```

Comment:
Hey, thanks for the commit. We already have instruction on how to use the tool in INSTALL.MD. Could you check with that doc and add these instructions there? We can also reference INSTALL.md from here.